multiview/cluster/mv_k_means.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from .base_cluster import BaseCluster
+from ..utils.cluster_utils import check_Xs
+
+
+class MultiviewKMeans(BaseCluster):
+
+    '''
+    An implementation of Multi-View K-Means using the co-EM algorithm.
+    This algorithm currently handles two views of data.
+
+    Paramters
+    ---------
+    k : int
+        The number of clusters
+
+    random_state : int (default=None)
+        Determines random number generation for initializing centroids.
+        Can seed the random number generator with an int.
+
+    max_iter : int (default=None)
```

Comment:
even though the default is technically none, I think this should probably say default = infinity because that's what the code changes it to

Reply:
I think I'll leave the default as none here since scikit-learn's API does the same for regular kmeans.